scripts/mrbind/generate.mk

```diff
@@ -64,7 +67,21 @@ $(error Must run this in Visual Studio developer command prompt, or at least cop
 endif
 endif
 
-# Windows-only vars: [
+# Set to 1 if you're planning to make a wheel from this module.
+FOR_WHEEL := 0
```

Comment:
should it be 1 by default?

Reply:
Decided to keep 0, since it's what's needed for local builds.